DESCRIPTION

```diff
@@ -46,7 +46,7 @@ Suggests:
     rlang,
     rmarkdown,
     rstudioapi (>= 0.2),
-    testthat (>= 3.2.1),
+    testthat (>= 3.3.0),
```

Comment:
we don't technically need this since it's GHA-only requirement right?

Reply:
We don't, but it's a good way to enforce the dev team to upgrade to use the latest version, which will become especially relevant if we run into a flaky test.

I know it's not ideal because they are forced to make this update globally, since we aren't using project dependency manager (like `{renv}`).